k8s/templates/values/values.yml

```diff
@@ -0,0 +1,7 @@
+#@data/values
+---
+version: "74.12.0"
+labels:
+  instance: standalone
+  partOf: uaa
```

Comment:
I'm assuming that it was an intentional choice not to have this be `Cloud Foundry`, even though that's what the [Idiomatic Components Doc](https://docs.google.com/document/d/1aVQkmCGmHMuCJS9h1UfO-THcP7ENdqIZe5lYtOzcPPA/edit?ts=5dc1b4ad#) suggests, because UAA is used by more products than just Cloud Foundry.

Reply:
It was.  Since the UAA is deployed to multiple environments I chose the base config to be standalone.  I assumed CF would provide value overrides for what's important to them.